pillar/credere_common.sls

```diff
@@ -1,57 +1,43 @@
-network:
-  host_id: ocp24
-  ipv4: 213.52.130.126
-  ipv6: "2a01:7e00:e000:06e6::"
-  networkd:
-    template: linode
-    gateway4: 213.52.130.1
-
 apache:
   public_access: True
   sites:
     credere:
       configuration: proxy
-      servername: ocp24.open-contracting.org
       # servername: credere.open-contracting.org
       context:
         proxypass: http://localhost:8000/
         # authname: Credere Staging
 
 docker:
   user: deployer
+  docker_compose:
+    version: v2.19.0
```

Comment:
This should only be on credere_dev (if even there). Docker Compose is now part of Docker itself, and the production server doesn't install it separately.